File: app/src/main/java/com/example/hw14app/App1.java
package com.example.hw14app;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class App1 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_app1);
    }

    public void onClick11(View v) {
        EditText mEditText = findViewById(R.id.editTextInput);
        String castToString = mEditText.getText().toString();
        TextView text = findViewById(R.id.textView5);

        castToString = castToString.replace('a', 'o');
        castToString = castToString.replace('A', 'O');

        text.setText(castToString);
    }
}

File: app/src/main/java/com/example/hw14app/App1.kt
package com.example.hw14app

import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class App1 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_app1)
    }

    fun convertLettersClick(view: View) {
        val mEditText = findViewById<EditText>(R.id.editTextInput)
        var castToString = mEditText.text.toString()
        val text = findViewById<TextView>(R.id.textView5)
        castToString = castToString.replace('a', 'o')
        castToString = castToString.replace('A', 'O')
        text.text = castToString
    }

}